docs/docs/how-to/performance/improving-site-performance.md

```diff
@@ -0,0 +1,203 @@
+- **Self-host rather than installing from an external CDN.** Having the font file available locally will save a trip over the network and reduce blocking time. 
+- **Use Latin font subsets only** (if creating a Latin-language site). It's common to accidentally include font extensions (Greek, Cyrillic, Devnagari, Chinese) when typically you only need the Latin base set. The [Google Webfonts Helper app](https://google-webfonts-helper.herokuapp.com/fonts/SourceSansPro) can help you do this with free fonts.
+
+Font optimizations are usually small, but easy performance wins. 
+
+### Images & Media
+
+Media files are often the largest files on a site, and so can delay page load significantly while they are pulled over the network, especially if their location is not well-defined.
+
+[Gatsby Image](/docs/how-to/images-and-media/using-gatsby-image/) is our approach to optimizing image loading performance. It does three basic things:
+
+- First, it delays non-essential work for images not above the fold to avoid esource congestion. 
+- Second, it provides a placeholder during image fetch.
+- Third, it minimizes image file size to reduce request roundtrip time.
+
+The gatsby-image documentation is fairly exhaustive, ranging from [why image optimization is important](/docs/conceptual/using-gatsby-image/), or [how to implement Gatsby Image](/docs/how-to/images-and-media/using-gatsby-plugin-image/), to a [Gatsby Image reference](/docs/reference/built-in-components/gatsby-image/).
+
+Implementing Gatsby Image is typically the bulk of image- and media-related performance optimization. 
+
+### Resource Requests & CDN Configuration
+
+Part of the work in loading a Gatsby site is minimizing the time to transport bits over the network. There's a number of ways to do this.
+
+First, load critical assets from your main domain where possible. Some people use another domain for their images. This can have a 300ms delay when it comes to LCP compared to loading it from the main CDN. This is sometimes necessitated by company policies; try to avoid it if possible.
+
+Second, preconnect to subdomains using [gatsby-plugin-preconnect](https://www.gatsbyjs.com/plugins/gatsby-plugin-preconnect/).
+
+Third, utilize Gatsby Link. Gatsby Link is our approach to optimizing the intra-site navigation  experience. We pre-load linked pages on your site so that transitioning between pages is smooth and seamless. [Here's a guide to using Gatsby Link](/docs/linking-between-pages/#the-gatsby-link-component).
+
+Fourth, implement proper CDN caching policies. Configuration differs on a per-CDN basis, but since Gatsby generates unique bundle names via hashes, bundles can be cached indefinitely since they are immutable. 
```

Comment:
Possibly make this a bullet list? Since they don't have to do them in a particular order.

Reply:
Good call.